### ISSUE TITLE
BatchNorm 与 LayerNorm | 各有未来

### ISSUE THREAD

**Issue by alyssaasa** [alyssaasa/alyssaasa.github.io#30]
https://alyssaasa.github.io/2019/11/08/LayerNorm-%E4%B8%8E-BatchNorm/ 

 🌼  Batch Normalization1. 作用：batch normalization 主要起到正则化的作用，可以加速网络收敛，并防止梯度消失。具体解释可以看： https://www.bilibili.com/video/av16000304 2. 做法： 对每个batch的元素求均值和方差：E(x) = \frac{1}{m}\sum\limits_i x_iVar(x) = \fr
